delivery_carrier_label_paazl/models/delivery_carrier_option.py

```diff
@@ -0,0 +1,13 @@
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class DeliveryCarrierOption(models.Model):
+    _inherit = 'delivery.carrier.option'
+
+    paazl_label_file_type = fields.Selection(
+        [("pdf", "PDF"), ("zpl", "ZPL")],
```

Comment:
Can we stick to upper case key names here, to be consistent with https://github.com/OCA/delivery-carrier/blob/12.0/base_delivery_carrier_label/models/carrier_account.py#L16-L18?

Reply:
Also stick to `file_format` terminology as used on the higher level rather than `file_type`.